agent/ui_endpoint_test.go

```diff
@@ -828,6 +828,7 @@ func TestUIGatewayIntentions(t *testing.T) {
 
 	a := NewTestAgent(t, "")
 	defer a.Shutdown()
+	testrpc.WaitForTestAgent(t, a.RPC, "dc1")
```

Comment:
Did you want to switch this to the new method I just added?

Reply:
Yep, done